build-system/config.js

```diff
@@ -126,6 +126,7 @@ module.exports = {
     '!build/polyfills/*.js',
     '!gulpfile.js',
     '!third_party/**/*.*',
+    '!validator/chromeextension/*.*',
```

Comment:
so we probably want to run the copyright presubmit check on these files. I'll add some code so that it is excluded except for that check.